app/javascript/dashboard/api/integrations/dyte.js

```diff
@@ -0,0 +1,23 @@
+/* global axios */
+
+import ApiClient from '../ApiClient';
+
+class DyteAPI extends ApiClient {
+  constructor() {
+    super('integrations/dyte', { accountScoped: true });
+  }
+
+  createAMeeting(conversationId) {
+    return axios.post(`${this.url}/create_a_meeting`, {
```

Comment:
Just curious, how is `axios` globally available? Is it a rails thing? Or we just attach axios to the window?

Reply:
I think it comes from this comment - `/* global axios */`

Reply:
Attached to the window object.

Reply:
Yes, it's a global window variable. Is there a better way to do this?
<img width="393" alt="image" src="https://user-images.githubusercontent.com/1277421/211254645-0c58b6e5-5d3a-409e-89ac-6b50c368c220.png">

Reply:
@nithindavid The API client should have taken the Axios object within it instead of making it a global variable. Essentially we are using axios in a similar way fetch works. 

This was on my list for refactoring, I couldn't get enough time to redo all the APIClients.

Reply:
> The API client should have taken the Axios object within it instead of making it a global variable.

This is ideal

> I couldn't get enough time to redo all the APIClients.

We can start by having axios proxied within the client and then slowly refactoring others, I can pick this up if you think it's worth doing